safe/definitions/hazard_classifications.py

```diff
@@ -607,12 +867,140 @@
     'exposures': [
         exposure_land_cover,
         exposure_place,
-        exposure_population,
         exposure_road,
         exposure_structure
     ]
 }
 
+# duplicate classes for tsunami hazard based on advice from Pak Hamza and
+# modified for population
+tsunami_hazard_classes = {
+    'key': 'tsunami_hazard_classes',
```

Comment:
this commit is the one where I need help from you to make it unique and fix all the other flow on issues. I am still not convinced that the way i have done this is a good way - it seems really clumsy to have so much duplication